### PR TITLE
fix(auto-instrumentations-node): add more instrumentations to getNodeAutoInstru…

### PR DESCRIPTION
**Pull request by haddasbronfman**
## Which problem is this PR solving?

The function getNodeAutoInstrumentations() does not return all the instrumentations, only part of them

## Short description of the changes

Add the missing instrumentations to the function getNodeAutoInstrumentations()

## Checklist

- [ ] Ran `npm run test-all-versions` for the edited package(s) on the latest commit if applicable.

### CODE REVIEW

**Comment by blumamir** [comment]
Hi @haddasbronfman ,
Thanks for the PR.
Can you please fix the conflicts and make sure your CLA works?
You can convert this PR to draft until it's ready for review.

Regarding the PR name - if you want to use a scope, please use something like the package name `auto-instrumentations-node` instead of `add-inst-to-dict`.
[link](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#specification):
> A scope MAY be provided after a type. A scope MUST consist of a noun describing a section of the codebase surrounded by parenthesis, e.g., fix(parser):

File: metapackages/auto-instrumentations-node/README.md [hunk 1]
```diff
@@ -55,18 +55,35 @@ registerInstrumentations({
...
+- [@opentelemetry/instrumentation-net](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-net)
+- [@opentelemetry/instrumentation-pg](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-pg)
+- [@opentelemetry/instrumentation-pino](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-pino)
+- [@opentelemetry/instrumentation-redis](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-redis)
+- [@opentelemetry/instrumentation-restify](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-restify)
+- [@opentelemetry/instrumentation-router](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-router)
+- [@opentelemetry/instrumentation-winston](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-winston)
```
Comment (blumamir) [inline metapackages/auto-instrumentations-node/README.md:86]: We can also add [instrumentation-tedious](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/instrumentation-tedious)

Comment (haddasbronfman) [reply metapackages/auto-instrumentations-node/README.md:86]: Thanks! I tried to add it by:
"npm install --save @opentelemetry/instrumentation-tedious"
and I got: 
"npm ERR! 404 Not Found - GET https://registry.npmjs.org/@opentelemetry%2finstrumentation-tedious - Not found"
Also, I couldn't find this in npm (I tried some searches..)

If you know how to solve this please let me know.

**Comment by haddasbronfman** [comment]
Close and reopen a new one

**Pull request closed without merging**